src/FinancialReports/FinancialSummaryByInsuranceController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Financial Report controller class.
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2016-2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Finance\Reports;   //This will function as my controller event though not named controller
+
+require_once "../../vendor/autoload.php";
```

Comment:
shouldn't need per above comment

Reply:
rearranged the folder structure for the 5.0.2/3 structure and was still able to get the template to display. This will serve as a base to build frontend displays rendered by twig.